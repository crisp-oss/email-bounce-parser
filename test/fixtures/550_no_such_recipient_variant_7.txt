This is the mail system at host mailer.acme.email.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<carol@gnail.com>: host mx1.oweb.cn[222.37.137.89] said: 550
    mailbox carol@gnail.com unavailable (???䲻�???) (in reply to
    RCPT TO command)
