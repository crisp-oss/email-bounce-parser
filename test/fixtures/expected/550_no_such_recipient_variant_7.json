{
  "bounce": true,
  "email": {
    "body": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system\n\n<carol@gnail.com>: host mx1.oweb.cn[222.37.137.89] said: 550 mailbox carol@gnail.com unavailable (???䲻�???) (in reply to RCPT TO command)",
    "intro": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system",
    "error": "<carol@gnail.com>: host mx1.oweb.cn[222.37.137.89] said: 550 mailbox carol@gnail.com unavailable (???䲻�???) (in reply to RCPT TO command)"
  },
  "data": {
    "error": {
      "code": {
        "basic": "550",
        "enhanced": null
      },
      "label": "mailbox carol@gnail.com unavailable (???䲻�???)",
      "type": "action_not_taken",
      "temporary": false,
      "permanent": true,
      "data": {
        "type": "no_such_recipient",
        "blocked": false,
        "spam": false
      }
    },
    "recipient": "carol@gnail.com",
    "server": {
      "hostname": "mx1.oweb.cn",
      "ip": "222.37.137.89",
      "port": null
    },
    "command": "RCPT TO"
  }
}